CharacterGenerator/CharacterGenerator/Bender.m

```diff
@@ -0,0 +1,89 @@
+//
+//  Bender.m
+//  CharacterGenerator
+//
+//  Created by Christian Maldonado on 8/4/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Bender.h"
+#import "BenderManager.h"
+#import "CharacterTableViewController.h"
+
+@implementation Bender
+
+-(void)animalForColor:(NSString *)color AndTexture:(NSString *)texture{
+    
+    
+    [self colorsAndTexturesToNumbers];
+    
+    NSString *c = [self.colorsToNumbers objectForKey:color];
+    NSString *t = [self.texturesToNumbers objectForKey:texture];
+    
+    NSInteger colorValue = [c integerValue];
```

Comment:
Since `c` is only used once, you can condense this to
`NSInteger colorValue = [[self.colorsToNumbers objectForKey:color] integerValue];`

Same for `t`.